allauth/socialaccount/providers/hubspot/provider.py

```diff
@@ -0,0 +1,29 @@
+from allauth.socialaccount.providers.base import ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class Scope(object):
+    oauth2 = 'oauth'
+
+
+class HubSpotAccount(ProviderAccount):
+    pass
+
+
+class HubSpotProvider(OAuth2Provider):
+    id = 'hubspot'
+    name = 'HubSpot'
+    account_class = HubSpotAccount
+
+    def extract_common_fields(self, data):
+        return dict(username=data.get('user'))
```

Comment:
The mocked response seems to contain a username and id, yet here you are using just `user` ? Is the mocked response not complete, or is this code out of sync?